apps/vs-code-designer/src/app/azuriteExtension/getAzuriteConfiguration.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { azuriteExtensionId } from '../../constants';
+import { localize } from '../../localize';
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+import { extensions } from 'vscode';
+import * as vscode from 'vscode';
+
+export async function getAzuriteConfiguration(context: IActionContext, configurationName: string): Promise<string> {
+  const azuriteExtension = extensions.getExtension(azuriteExtensionId);
+
+  if (azuriteExtension?.isActive) {
+    const configuration = vscode.workspace.getConfiguration();
```

Comment:
We have a getWorkspaceSetting function in here apps/vs-code-designer/src/app/utils/vsCodeConfig/settings.ts. Its up to you if you want to call the function in here or delete the file and call the function directly where needed